pkg/pillar/evetpm/encryptdecypt.go

```diff
@@ -210,3 +200,13 @@ func EncryptDecryptUsingTpm(in []byte, encrypt bool) ([]byte, error) {
 	}
 	return out, err
 }
+
+// ECC coordinates need to maintain a specific size based on the curve, so we pad the front with zeros.
```

Comment:
You have a yetus thing to fix on this line as shown in the annotation in https://github.com/lf-edge/eve/pull/4908/files